data/json/construction.json

```diff
@@ -2204,5 +2204,19 @@
    ],
    "pre_special" : "check_empty",
    "post_terrain" : "f_forge_rock"
+},{
+   "type" : "construction",
+   "description" : "Build Clay Kiln",
+   "category" : "FURN",
+   "difficulty" : 4,
+   "time" : 40,
+   "qualities": [
+   [ {"id": "HAMMER", "level": 2 } ]
+   ],
+   "components" : [
+      [ [ "rock", 40 ] ]
```

Comment:
I'd agree, but you would need a crucible to make cement, which requires a steel frame for making it. Probably, use some raw clay instead?